Makefile

```diff
@@ -249,32 +249,29 @@ install-travis:
 # Build RPMs only for the Linux AMD64 target
 #
 # Args:
-#   BUILD_TESTS: whether or not to build a test RPM, off by default
 #
 # Example:
 #   make build-rpms
 build-rpms:
-	BUILD_TESTS=$(BUILD_TESTS) OS_ONLY_BUILD_PLATFORMS='linux/amd64' hack/build-rpm-release.sh
```

Comment:
That's amazing. Should also remove it from `origin.spec` then, it was only there to support us